earth_enterprise/rpmlintrc

```diff
@@ -0,0 +1,41 @@
+# -- revisit for later review
+
+# Disable to review/remediate later...
+addFilter("E: non-standard-executable-perm")
+
+# Disable to ignore for now, often issues from third party code, some is valid though.
+addFilter("^.* /opt/")
+addFilter("^.* /var/opt/")
+
+# Because this is in init.d it gets LOTS of error hits...
+addFilter("E: .* /etc/init.d/gevars.sh")
+
+# We are not currently renaming the logrotate file...
+addFilter("E: incoherent-logrotate-file")
+
+# We do not currently do changelogs in rpm specfiles...
+addFilter("E: no-changelogname-tag")
+
+# We should fix this...
+addFilter("E: devel-dependency proj-devel")
```

Comment:
Yeah, why add a filter, if we need to fix it?

Reply:
I know we had problems removing this before....

Reply:
OK.  We could use a ticket for fixing that, then.

It may still be useful to see the errors that we need to fix.

Reply:
The purpose was to be able to use lint to verify no "new" errors are being introduced with future builds, hence why they are filtered, and any error halts the target.  The important thing to note, yes, is that those issues that are not being remediated now are documented and should be addressed in separate issues.  Some potential remediations I was cautious about because I did not want to break builds, and I thought they would be better addressed as stand-alone issues on their own.

Reply:
That makes perfect sense.  Writing tickets to fix these is fine.